librad/src/net/protocol/accept.rs

```diff
@@ -79,20 +79,28 @@ where
                     .into(),
                 }])
             },
+
+            membership::Periodic::Tickle => {
+                // Tickle connections in the partial view.
+                //
+                // This is mostly to keep passive connections from being collected. Note
+                // that we're not checking actual liveness, nor interfering with the
+                // membership protocol.
+                tracing::debug!("initiating tickle");
+                for peer in state.membership.known() {
+                    if let Some(conn) = state.endpoint.get_connection(peer) {
+                        conn.tickle();
+                    }
+                }
+
+                // FIXME(xla): This indicates that while tickling is a periodic task this method
```

Comment:
This is fine. If it looks funny to you, then because `accept` also _evaluates_ the tocks. More cleansily, this could be two functions, one which transforms the periodic events into tocks, and another one feeds them to the tock-evaluator.